--- STM32F407VGTx_FLASH.ld	2024-11-30 23:31:09.451123790 +0400
+++ STM32F407VGTx_FLASH.ld.fix	2024-11-30 23:32:31.563122867 +0400
@@ -95,6 +95,7 @@
   {
     . = ALIGN(4);
     _sdata = .;        /* create a global symbol at data start */
+    *(.ramfunc*)       /* .ramfunc sections */
     *(.data)           /* .data sections */
     *(.data*)          /* .data* sections */
 
@@ -121,7 +122,6 @@
     _eccmram = .;       /* create a global symbol at ccmram end */
   } >CCMRAM AT> FLASH
 
-  
   /* Uninitialized data section */
   . = ALIGN(4);
   .bss :
@@ -149,8 +149,6 @@
     . = ALIGN(8);
   } >RAM
 
-  
-
   /* Remove information from the standard libraries */
   /DISCARD/ :
   {
@@ -158,5 +156,4 @@
     libm.a ( * )
     libgcc.a ( * )
   }
-
 }
