--- STM32F407VGTx_FLASH.ld	2024-11-30 23:26:11.291127141 +0400
+++ STM32F407VGTx_FLASH.ld.fix	2024-11-30 23:27:59.219125928 +0400
@@ -122,7 +122,6 @@
     _eccmram = .;       /* create a global symbol at ccmram end */
   } >CCMRAM AT> FLASH
 
-  
   /* Uninitialized data section */
   . = ALIGN(4);
   .bss :
@@ -150,8 +149,6 @@
     . = ALIGN(8);
   } >RAM
 
-  
-
   /* Remove information from the standard libraries */
   /DISCARD/ :
   {
@@ -159,5 +156,4 @@
     libm.a ( * )
     libgcc.a ( * )
   }
-
 }
