--- STM32F407VGTx_FLASH.ld	2024-11-25 16:18:49.653198029 +0400
+++ STM32F407VGTx_FLASH.ld.fix	2024-11-25 16:19:17.441197717 +0400
@@ -102,6 +102,7 @@
     _sdata = .;        /* create a global symbol at data start */
     *(.data)           /* .data sections */
     *(.data*)          /* .data* sections */
+    *(.ramfunc*)       /* .ramfunc sections */
 
     . = ALIGN(4);
     _edata = .;        /* define a global symbol at data end */
